src/analyses/constant_propagator.cpp

```diff
@@ -93,7 +93,9 @@ void constant_propagator_domaint::transform(
     ignore_unresolved_calls=cp->ignore_unresolved_calls;
   }
 
-  assert(!values.is_bottom);
+  //assert(!values.is_bottom);
+  if(values.is_bottom)
+    return;
```

Comment:
As far as I'm aware this reverts a change introduced in this commit series - would you mind squashing these changes?